app/assets/javascripts/map/views/layers/GladCoverageLayer.js

```diff
@@ -1,13 +1,13 @@
 define([
-  'abstract/layer/CartoDBLayerClass',
-], function(CartoDBLayerClass) {
+  'abstract/layer/CartoDBLayerClass'
+], (CartoDBLayerClass) => {
```

Comment:
This breaks IE. Can you revert to the old syntax?

Reply:
Mustve been linted